wallet/datastore.go

```diff
@@ -559,20 +559,32 @@ func (pg *Postgres) IncreaseLinkingLimit(ctx context.Context, providerLinkingID
 
 // LinkWallet links a wallet together
 func (pg *Postgres) LinkWallet(ctx context.Context, ID string, userDepositDestination string, providerLinkingID uuid.UUID, anonymousAddress *uuid.UUID, depositProvider string) error {
-	sublogger := logger(ctx).With().
-		Str("wallet_id", ID).
-		Logger()
+	sublogger := logger(ctx).With().Str("wallet_id", ID).Logger()
 
 	// create tx
 	tx, err := createTx(ctx, pg)
```

Comment:
https://github.com/brave-intl/bat-go/blob/master/wallet/datastore.go#L384-L390 switch to using getTx.  Feel like it is a better model.